src/vorta/store/settings.py

```diff
@@ -59,6 +59,18 @@ def get_misc_settings() -> List[Dict[str, str]]:
             'label': trans_late('settings', 'Get statistics of file/folder when added'),
             'tooltip': trans_late('settings', 'When adding a new source, calculate its size and the number of files.'),
         },
+        {
+            'key': 'enable_fixed_units',
+            'value': False,
+            'type': 'checkbox',
+            'group': information,
+            'label': trans_late('settings', 'Use the same unit of measurement for archive sizes'),
+            'tooltip': trans_late(
+                'settings',
+                'When enabled, all archive sizes will use the same unit of measurement, '
+                'such as  KB or MB. This can make archive sizes easier to compare.',
```

Comment:
I used the word "compare" instead of "read", since that might to be closer to the original intention of the setting.

Reply:
I like that!